applicationset/controllers/applicationset_controller.go

```diff
@@ -155,6 +155,7 @@ func (r *ApplicationSetReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 	// desiredApplications is the main list of all expected Applications from all generators in this appset.
 	desiredApplications, applicationSetReason, err := template.GenerateApplications(logCtx, applicationSetInfo, r.Generators, r.Renderer, r.Client)
 	if err != nil {
+		logCtx.Errorf("unable to gererate applications: %v", err)
```

Comment:
Minor nit, otherwise LGTM
```suggestion
		logCtx.Errorf("unable to generate applications: %v", err)
```